name_mappings.json

```diff
@@ -13,7 +13,7 @@
     },
     "fedora-commops": {
```

Comment:
This should also be updated to `commops`, and `commops` should be removed from the aliases here`